src/components/styles.js

```diff
@@ -165,6 +165,26 @@ const styles = theme => ({
     height: 'auto',
     marginTop: '15px',
     padding: '15px'
+  },
+
+  // Loading Styling
+  loading_wrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: '10%'
+  },
+
+  // emptyState Styling
```

Comment:
Can we change this comment so instead of just saying `emptyState`, we describe what it is. Like `// Image Wrapper styling or Image Container Styling`